aea/configurations/constants.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2020 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Module to declare constants."""
+
+from pathlib import Path
+
+from aea.configurations.base import PublicId
+from aea.crypto.fetchai import FETCHAI
+
+DEFAULT_CONNECTION = PublicId.from_str("fetchai/stub:0.1.0")
```

Comment:
All public ids should be specified in string form, makes it easier to do string substitution

Reply:
yes.